From a756a9df13b8dabbdc7250d500435004fa1332e7 Mon Sep 17 00:00:00 2001
From: "tao.zeng" <tao.zeng@amlogic.com>
Date: Wed, 27 Nov 2013 14:12:31 +0800
Subject: [PATCH 2088/5965] PD #83187: Reduce max backlight brigtness level to
 save power consumption

Backligt brightness of K100 increase from 0 to 255 will cause 1.4A current
increase of battery, when system is running large loading 3D game, battery
current will increase to more than 4A @ max backlight brightness level. This
value exceed max current measure range of PMU, will have bad effect for
power supply subsystem. So need to reduce max backlight brigtness level for
K100
---
 arch/arm/boot/dts/amlogic/meson8_k100_v1.dtd | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/amlogic/meson8_k100_v1.dtd b/arch/arm/boot/dts/amlogic/meson8_k100_v1.dtd
index e815875a3f56..f73464385c6e 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k100_v1.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k100_v1.dtd
@@ -834,7 +834,7 @@ sdio{
 		bl_gpio_dim_max_min=<0x0 0xf>;	/** brightness diming level_max, level_min : (0x0~0xf, negative logic, only for gpio control method) */
 		bl_pwm_port_gpio_used="PWM_C","1";	/** pwm_port_name(for pwm control method), pwm_gpio_used(0=use pwm_port only, 1=use bl_gpio_port to control on/off) */
 		bl_pwm_freq=<1000>;	/** backlight control pwm frequency(unit: Hz, only for pwm control method) */
-		bl_pwm_duty_max_min=<100 20>;	/** brightness diminig duty_max, duty_min : (unit: %, positive logic, only for pwm control method) */
+		bl_pwm_duty_max_min=<85 20>;	/** brightness diminig duty_max, duty_min : (unit: %, positive logic, only for pwm control method) */
 	};
 //$$ DEVICE="lcd"
 //$$ L2 PROP_STR = "status"
-- 
2.19.0

